index.html

```diff
@@ -104,8 +104,14 @@ <h2 class="inter-font">Gain+Glory</h2>
             <li><button type="button" class="inter-font">javascript</button></li>
             <li><button type="button" class="inter-font">html</button></li>
         </ul>
-        <button class="seeProject inter-font" type="button">See Project</button>
+        <button class="seeProject inter-font" type="button" id="seeProjectPopup" >See Project</button>
```

Comment:
- [ ] Kindly remove the hard code projects and instead load the projects dynamically using javascript, you have already created an array in `popupWindow.js`, you can check this examples for more understanding: [example 1 ](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [example 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96) .